src/graphql/data/nft/Collection.ts

```diff
@@ -1,10 +1,9 @@
 import graphql from 'babel-plugin-relay/macro'
 import { GenieCollection, Trait } from 'nft/types'
-import { useEffect } from 'react'
-import { useLazyLoadQuery, useQueryLoader } from 'react-relay'
 
-import { CollectionQuery } from './__generated__/CollectionQuery.graphql'
+import { useCollectionQueryQuery } from '../__generated__/types-and-hooks'
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const collectionQuery = graphql`
```

Comment:
By using the generated Apollo hook for 1useCollectionQueryQuery1, I don't need to directly refer to the gql query as compared to doing `useQuery<CollectionQuery>` which causes a linting error. I'm taking any suggestions for a cleaner solution here.

Reply:
one option is to could move to a separate `gql` file

Reply:
thoughts @zzmp ? I think I prefer colocating the query in the respective file